src/library/Table/styled.js

```diff
@@ -80,6 +80,7 @@ export const TableOverflowContainer = themed(OverflowContainer)(
 export const TableRoot = styled('table')(({ theme }) => ({
   ...componentStyleReset(theme),
 
+  backgroundColor: 'white',
```

Comment:
This is a bad assumption. What happens if we do want it to be transparent or light gray? Let's not hardcode this. Instead, provide a passthrough prop.

Let's remove this change from this PR since it's out of the scope of supporting fullWidth/width. Open up another PR to support setting backgroundColor. I'm okay with making "white" the default, but we need to support making it transparent or any other color.